Encountify/Views/ItemsPage.xaml.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Encountify.ViewModels;
```

Comment:
If I understand correctly then ItemsPage is Page with LocationList. If that's the case I believe it should be rename to more clear name.

Reply:
Sure thing!